integration-cli/docker_cli_plugins_test.go

```diff
@@ -169,7 +169,12 @@ func (ps *DockerPluginSuite) TestPluginSet(c *check.C) {
 
 	initialValue := "0"
 	err = plugin.Create(ctx, client, name, func(cfg *plugin.Config) {
-		cfg.Env = []types.PluginEnv{{Name: "DEBUG", Value: &initialValue, Settable: []string{"value"}}}
+		cfg.Env = []types.PluginEnv{
+			{Name: "DEBUG", Value: &initialValue, Settable: []string{"value"}},
+		}
+		cfg.Mounts = []types.PluginMount{
```

Comment:
Similar to this, please add a `Devices` config with a nil `Path` and ensure that test passes on inspect